concepticondata/conceptlists.tsv

```diff
@@ -167,3 +167,4 @@ Syrjaenen-2013-226	Syrjänen, Kaj and Honkola, Terhi and Korhonen, Kalle and Leh
 Wheeler-2015-102	Wheeler, W. C. and Whiteley, Peter M.	2015		102	basic	English	Uto-Aztekan	http://onlinelibrary.wiley.com/doi/10.1111/cla.12078/suppinfo	Wheeler2015		This list contains 102 concept labels which were used for automatic analyses on Uto-Aztekan languages. They state in their paper that they use a version of [Swadesh's list of 100 items](:ref:Swadesh-1955-100), yet they add two more items, since they split &quot;to kill&quot; into two concepts (singular versus plural), and also &quot;earth&quot;, thereby distinguishing between &quot;earth as soil&quot; and &quot;earth as mud&quot;. The data is not shown in their paper but was taken from the supplementary material.		
 Janhunan-2008-235	Janhunan, Juha and Peltomaa, Marja and Sandman, Erika and Dongzhou, Xiawu	2008		273	areal	English	Wutun		Janhunan2008	Janhunan2008	This list was used to document basic vocabulary of the Wutun language, an apparently Sinitic language spoken in the Amdo region of Tibet. The numbering of the word list is rather strange, since it is either inconsistent, or there is a system that is not directly understandable for the reader. The authors list general terms which we know from normal basic vocabulary lists, but also cultural variants, yet apart from that, they also list items which are not given a further translation (and which we therefore had to leave unmapped).	118-127	
 Dunn-2011-146	Dunn, Michael and Burenhult, Niclas and Kruspe, Nicole and Tufvesson, Sylvia and Becker, Neele 	2011		146	areal	English	Aslian	http://www.jbe-platform.com/content/journals/10.1075/dia.28.3.01dun	Dunn2011		This list was created by Niclas Burenhult and Nicole Kruspe in order to document the Aslian languages of the highlands of the Malay Peninsula. This is based on the Swadesh list, but also includes some relevant culture/technology words.	291-323	
+Daniels-2016-85	Daniels, Don	2016		85	basic	English	Magi,Aisi	http://dx.doi.org/10.1353/ol.2016.0004	Daniels2016	Daniels2016	This is a Swadesh 100 list used to document two languages of New Guinea	219-221	
```

Comment:
I would suggest to re-name the description with additional link:

```text
This is a derived version of [Swadesh's list of 100 items (1955)](:ref:Swadesh-1955-100) used to document two languages of New Guinea.
```

Instead of labelling only as basic, I would suggest to label as "basic,areal", as I assume the missing concepts result from areal patterns in NG.